app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,75 @@
+!!!
+%html
+  %head
+    %title Edu
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+  %body
+    .wrapper
+      %header.navbar.navbar-default.navbar-fixed-top.navbar-top
+        .container.container-fluid
+          .navbar-header
+            %button.navbar-toggle{"data-target" => ".navbar-collapse", "data-toggle" => "collapse", type: "button"}
+              %span.sr-only Toggle navigation
+              %span.icon-bar
+              %span.icon-bar
+              %span.icon-bar
+            %a.navbar-brand{href: "index.html"}
+              %span.logo
+                %i.fa.fa-graduation-cap
+                Coursotheca
+          .navbar-collapse.collapse
+            %ul.nav.navbar-nav.navbar-right
+              %li.new-ads
+                %a.btn.btn-ads.btn-block{href: "account_create_post.html"}= I18n.t('labels.courses_list')
+              - if current_user
+                %li
+                  %a{href: "/profile"}= I18n.t('labels.to_profile')
+              - else
+                %li
+                  %a{href: "signup.html"}= I18n.t('labels.signup')
+              %li.dropdown
+                %a.dropdown-toggle{"data-toggle" => "dropdown", href: "#"}
+                  %strong.caret
+                  = I18n.t('labels.pages')
+                %ul.dropdown-menu
+                  %li
+                    %a{href: "/about"}= I18n.t('labels.about')
+              %li.dropdown
+                %a.dropdown-toggle{"data-toggle" => "dropdown", href: "#"}
+                  %i.fa.fa-user
+                  = succeed "\u{a0}" do
```

Comment:
Appends a string to the end of a Haml block, with no whitespace between. For example:

click
= succeed '.' do
  %a{:href=>"thing"} here
Produces:

click
<a href='thing'>here</a>.